l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -353,6 +353,28 @@ def _prepare_invoice(self):
         })
         return res
 
+    @api.multi
+    def other_operations_on_ddt(self, invoice):
+        """ Once invoices are created with stockable products, we add them
+        all the invoiceable services available in the SO related to the
+        DDTs linked to the invoice.
+
+        Override this method in order to execute other additional operation on
+        the invoices created from DDT.
+        """
+        for ddt in self:
+            order_ids = ddt.line_ids.mapped('sale_line_id.order_id').filtered(
+                lambda o: not o.ddt_invoice_exclude)
+            line_ids = order_ids.order_line.filtered(
```

Comment:
```suggestion
            line_ids = order_ids.mapped('order_line').filtered(
```
`order_ids.order_line` dà errore se `order_ids` contiene diversi ordini.